core/src/main/res/values-night/themes.xml

```diff
@@ -4,6 +4,17 @@
   <!--Top level DayNight theme to be used in AndroidManifest.xml-->
   <style name="KiwixTheme" parent="Base.KiwixTheme">
     <!-- Dark theme customisations here-->
+    <item name="colorPrimary">@color/denim_blue400</item>
+    <item name="colorSurface">@color/mine_shaft_gray900</item>
+    <item name="colorError">@color/froly_pink300</item>
+
+
+    <item name="colorOnPrimary">@color/alabaster_white</item>
+    <item name="colorOnSurface">@color/alabaster_white</item>
+
+    <!--Black has a contrast ratio of 6.9:1 vs
+        white which has a ratio of 3.05:1 (on froly_pink300)-->
+    <item name="colorOnError">@color/black</item>
```

Comment:
black on red? That sounds very bad to the eye

Reply:
This will only ever be used in a button so will be all caps and our default button text size is 14sp so I think we are within [accessibility margins](https://material.io/design/usability/accessibility.html#color-and-contrast)
![image](https://user-images.githubusercontent.com/3358282/85742018-d03d9f00-b6fa-11ea-9793-7e560587e8f2.png)

also I think the white text aligns this much more closely with our other buttons, this would stick out I think

Reply:
Good points. I agree.